src/components/Nav/Nav.jsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useHistory } from 'react-router';
 import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
 function Nav() {
+  const [log, setLog] = useState(false);
+  const [color, setColor] = useState({ color: 'grey' });
```

Comment:
- 여기도 color 라는 이름이 어떤 color 를 말하는 건지 파악하기 어렵습니다. 명확한 네이밍으로 수정해주세요!
- state 의 데이터 구조 또한 객체로 구성해주실 필요가 없어보입니다. 
```suggestion
  const [color, setColor] = useState('grey');
```
- 그런데 color 스테이트는 아무 곳에서도 사용하고 있지 않은 것 같습니다. 확인해 주세요! 만약 사용하지 않는 스테이트라면 삭제해 주세요!